examples/inabox.host.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <title>amp-inabox-host example</title>
+</head>
+<body>
+  <div style="height: 120vh"></div>
+  <iframe
+      src="http://ads.localhost:8000/examples/inabox.amp.max.html"
+      scrolling="no"
+      width="300" height="200">
+  </iframe>
+  <div style="height: 120vh"></div>
+
+  <script>
+    initInaboxHost(window);
+
+    function initInaboxHost(win) {
+      const hostScriptUrl = 'http://localhost:8000/dist/amp-inabox-host.js';
+      preloadScript(win, hostScriptUrl);
+
+      // Keep all iframes that potentially can render AMP content in this array
+      win.ampInaboxIframes = [];
```

Comment:
because the host script is loaded later. we want to capture any inabox message before it gets loaded.